### PR TITLE
Updating resources section on Homepage

### PR DESCRIPTION
**Pull request by Santosh1176**
Signed-off-by: Santosh Kaluskar <dtshbl@gmail.com>

Updates the resources section on the homepage with updated YouTube resources.
This PR would resolve issue [#941](https://github.com/fluxcd/website/issues/941#issue-1236770970)

### CODE REVIEW

**Comment by dholbach** [comment]
https://deploy-preview-1046--fluxcd.netlify.app/ looks great to me.

@staceypotter What do you think?

File: content/en/_index.html [hunk 1]
```diff
@@ -204,41 +204,41 @@ <h2 class="section-label">Resources</h2>
...
 
   {{% blocks/resource
-    url="https://www.youtube.com/playlist?list=PLG9qZAczREKmCq6on_LG8D0uiHMx1h3yn"
-    thumbnail="N6UCKF7JD7k"
-    title="Flux Deep Dive series with Geert Baeke" %}}
-  In this 5 video series, Geert Baeke takes a deep dive of Flux and the use of GitOps principles with an Intro to Flux, Intro to Kustomize, Deploying Manifests, Monitoring & Alerting, and Helm Basics.
+    url="https://www.youtube.com/playlist?list=PL9lTuCFNLaD0NVkR17tno4X6BkxsbZZfr"
+    thumbnail="qCDWsIcFU-A"
+    title="GitOps Days 2022" %}}
+  A playlist with a compilation of all talks during GitOps Days 2022 on June 8-9, 2022.
```
Comment (staceypotter) [inline content/en/_index.html:241]: WDYT about this instead:
Playlist from GitOps Days 2022 on June 8-9, 2022 featuring Flux integrators, end-users/adopters, and maintainers.

**Review by staceypotter — CHANGES_REQUESTED** [review]
Made some suggestions for the descriptions on some of these. Thanks so much for this @Santosh1176 and great job!

**Comment by Santosh1176** [comment]
@staceypotter Thanks for the suggestions, I've updated the description accordingly.

**Review by dholbach — APPROVED** [review]
Thanks a lot @Santosh1176

**Pull request merged**